src/Uno.UI/DataBinding/BindingPropertyHelper.FastConvert.cs

```diff
@@ -53,10 +53,10 @@ private static bool FastConvert(Type outputType, object input, out object output
 		{
 			output = null;
 
-			if (
-				input is string stringInput
-				&& FastStringConvert(outputType, stringInput, ref output)
-			)
+			if (input is string stringInput && (
+				FastNumericalEnumConvert(outputType, stringInput, ref output) ||
```

Comment:
Shouldn't we do FactNumericalEnumConvert after FastStringConvert - I'd have assumed it more common to use the string value of an enum, rather than the value. By putting the numeric convert first, it's going to be run everytime, instead of only as a backup